ansible_builder/utils.py

```diff
@@ -82,7 +82,7 @@ def run_command(command, capture_output=False, allow_error=False):
         logger.debug(line.rstrip('\n'))  # line ends added by logger itself
     logger.debug('')
 
-    rc = process.poll()
+    rc = process.wait()
```

Comment:
@shanemcd this here is important and I want to get you to weigh in.

This fixes the `test_failed_command` tests being added in this PR, for what it's worth.

Reply:
Wow! Incredible. 💯